optuna/study.py

```diff
@@ -322,6 +322,25 @@ def _should_stop() -> bool:
     def set_user_attr(self, key: str, value: Any) -> None:
         """Set a user attribute to the study.
 
+        Example:
+
+            .. testcode::
+
+                import optuna
+
+                def objective(trial):
+                    x = trial.suggest_float("x", 0, 1)
+                    y = trial.suggest_float("y", 0, 1)
+                    return x ** 2 + y ** 2
+
+                study = optuna.create_study()
+
+                study.set_user_attr("objective function", "quadratic fuction")
+                study.set_user_attr("dimensions", 2)
+                study.set_user_attr("contributors", ["Akiba", "Sano"])
+
+                study.user_attrs
```

Comment:
As the same as here: https://github.com/optuna/optuna/pull/1745#discussion_r479732808, how about checking the contents?

Reply:
Thanks!